app/models/package_manager/pypi.rb

```diff
@@ -127,7 +127,7 @@ def self.dependencies(name, version, _mapped_project)
       api_response = get("https://pypi.org/pypi/#{name}/#{version}/json")
       deps = api_response.dig("info", "requires_dist")
       source_info = api_response.dig("releases", version)
-      Rails.logger.warn("Pypi sdist (no deps): #{name}") unless source_info.any? { |rel| rel["packagetype"] == "bdist_wheel" }
+      #Rails.logger.warn("Pypi sdist (no deps): #{name}") unless source_info.any? { |rel| rel["packagetype"] == "bdist_wheel" }
```

Comment:
This might be nicer, to maintain the warnings:

```suggestion
      Rails.logger.warn("Pypi sdist (no deps): #{name}") unless source_info&.any? { |rel| rel["packagetype"] == "bdist_wheel" }
```

Reply:
Confirmed locally that this behaves the same as commenting it out.